packages/blockchain-wallet-v4/src/exchange/delegate/index.js

```diff
@@ -86,10 +86,9 @@ export class ExchangeDelegate {
   }
 
   reserveReceiveAddress () {
-    // const receiveAddress = btc.getNextAvailableReceiveAddress(Bitcoin.networks.bitcoin.NETWORK_BITCOIN, 0, this.state)
+    const receiveAddress = btc.getNextAvailableReceiveAddress(Bitcoin.networks.bitcoin.NETWORK_BITCOIN, 0, this.state)
```

Comment:
Sorry for just noticing this, but this'll always pick your first account, even if default account is index > 0

Reply:
I think we can import selectors `core.wallet.getDefaultAccountIndex(this.state)`

Reply:
Same issue here actually: https://github.com/blockchain/blockchain-wallet-v4-frontend/blob/a6a1bd83de77c43370a8348d7cb82b121052a594/packages/blockchain-wallet-v4-frontend/src/scenes/BuySell/SfoxCheckout/Content/selectors.js#L9

Reply:
Good call. Fixed.